src/components/Item/Item.scss

```diff
@@ -0,0 +1,19 @@
+@import '../../variables.scss';
+
+.tlbx-item-code {
+
+  margin: 2rem 0 0;
+}
+
+.tlbx-item-code pre {
+  padding: 0.6rem;
+  box-shadow: inset 0 0 0 2px rgba(255,255,255,0.4);
+
+  // Code sample
+  code:last-child {
+    // Hide last empty line (:empty not working, because React)
+    > span:last-of-type {
```

Comment:
Be careful, maybe add a hint on the doc about last empty line on every `.twig` file.